lib/Arch/Runtime/HyperCall.cpp

```diff
@@ -0,0 +1,310 @@
+      break;
+
+    case SyncHyperCall::kAArch32CheckNotEL2:
+      mem = __remill_aarch32_check_not_el2(mem);
+      break;
+
+#elif REMILL_HYPERCALL_AARCH64
+
+    case SyncHyperCall::kAArch64EmulateInstruction:
+      mem = __remill_aarch64_emulate_instruction(mem);
+      break;
+
+    case SyncHyperCall::kAArch64Breakpoint: asm volatile("bkpt" :); break;
+
+#elif REMILL_HYPERCALL_SPARC32 || REMILL_HYPERCALL_SPARC64
+
+    case SyncHyperCall::kSPARCSetAsiRegister:
+      mem = __remill_sparc_set_asi_register(mem);
+      break;
+
+    case SyncHyperCall::kSPARCUnimplementedInstruction:
+      mem = __remill_sparc_unimplemented_instruction(mem);
+      break;
+
+    case SyncHyperCall::kSPARCUnhandledDCTI:
+      mem = __remill_sparc_unhandled_dcti(mem);
+      break;
+
+    case SyncHyperCall::kSPARCWindowUnderflow:
+      mem = __remill_sparc_window_underflow(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondA:
+      mem = __remill_sparc_trap_cond_a(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondN:
+      mem = __remill_sparc_trap_cond_n(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondNE:
+      mem = __remill_sparc_trap_cond_ne(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondE:
+      mem = __remill_sparc_trap_cond_e(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondG:
+      mem = __remill_sparc_trap_cond_g(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondLE:
+      mem = __remill_sparc_trap_cond_le(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondGE:
+      mem = __remill_sparc_trap_cond_ge(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondL:
+      mem = __remill_sparc_trap_cond_l(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondGU:
+      mem = __remill_sparc_trap_cond_gu(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondLEU:
+      mem = __remill_sparc_trap_cond_leu(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondCC:
+      mem = __remill_sparc_trap_cond_cc(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondCS:
+      mem = __remill_sparc_trap_cond_cs(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondPOS:
+      mem = __remill_sparc_trap_cond_pos(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondNEG:
+      mem = __remill_sparc_trap_cond_neg(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondVC:
+      mem = __remill_sparc_trap_cond_vc(mem);
+      break;
+
+    case SyncHyperCall::kSPARCTrapCondVS:
+      mem = __remill_sparc_trap_cond_vs(mem);
+      break;
+
+#  if defined(REMILL_HYPERCALL_SPARC32)
+
+    case SyncHyperCall::kSPARC32EmulateInstruction:
+      mem = __remill_sparc32_emulate_instruction(mem);
+      break;
+
+#  elif defined(REMILL_HYPERCALL_SPARC64)
+
+    case SyncHyperCall::kSPARC64EmulateInstruction:
+      mem = __remill_sparc64_emulate_instruction(mem);
+      break;
+
+#  endif
+
+#endif
+
+    default: break;
```

Comment:
Unfortunately, I can't `abort` here anymore since I don't have access to the standard library.

Reply:
[b93978a](https://github.com/lifting-bits/remill/pull/611/commits/b93978abd9729b5402e677c0f411a459d2a6a054)